src/Bicep.Cli/Extensions/IOContextExtensions.cs

```diff
@@ -5,9 +5,14 @@ namespace Bicep.Cli.Extensions
 {
     public static class IOContextExtensions
     {
+        public static void WriteCommandDeprecationWarning(this IOContext io, string deprecatingCommand, string newCommand)
+        {
+            io.Error.WriteLine($"DEPRECATED: The command {deprecatingCommand} is deprecated and will be removed in a future version of Bicpe CLI. Use {newCommand} instead.");
```

Comment:
Is it expected that a deprecated command show an error instead of a warning?  Does the command return 0 or 1 for command line?

Reply:
Yes, it's expected. We write warnings to stderr.

Reply:
The command returns 0.